pkg/apis/org/v1/che_types.go

```diff
@@ -410,7 +410,7 @@ type CheClusterSpecAuth struct {
 	// and have their workspaces created under personal OpenShift namespaces.
 	// WARNING: the `kubeadmin` user is NOT supported, and logging through it will NOT allow accessing the Che Dashboard.
 	// +optional
-	OpenShiftoAuth bool `json:"openShiftoAuth"`
+	OpenShiftoAuth *bool `json:"openShiftoAuth"`
```

Comment:
We have to update description to fit the behavior.

Reply:
Ok, I will